packages/styling/src/styles/getFocusStyle.ts

```diff
@@ -40,4 +40,22 @@ export function getFocusStyle(
 
     }
   };
+}
+
+/**
+ * Generates style to clear browser specific focus styles.
+ */
+export function focusClear(): IRawStyle {
```

Comment:
focusClear is a mixin used by several components so I created a function in the styling package. I just realized that there is one in the experiments package. What are your thoughts? Should I remove the one from the experimental package if we keep this hear?